File: Bank/EntitiesVb/Bank.vb
Public Class Bank
    Private _customers As List(Of Customer)
    Public Sub New()
        _customers = New List(Of Customer)
    End Sub
    Public Sub AddCustomer(customer As Customer)
        _customers.Add(customer)
    End Sub
    Public Function GetCustomers() As List(Of Customer)
        Return _customers
    End Function
End Class

File: Bank/EntitiesCs/Bank.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EntitiesCs
{
    public class Bank
    {
        private List<Customer> customers;
        public Bank()
        {
            customers = new();
        }
        public void AddCustomer(Customer customer)
        {
            customers.Add(customer);
        }
        public List<Customer> GetCustomers()
        {
            return customers;
        }
    }
}
